charts/functions/values.yaml

```diff
@@ -32,8 +32,6 @@ functions:
       max: 5
     timeout: 10s
     envvars:
-      STORAGE_CONTAINER_NAME: uploads
-      STORAGE_CONNECTION_STRING_SECRET: value
```

Comment:
out of sheer curiosity, why are these two no longer needed?

Reply:
They should have been removed when we added the file download to the Document Service. This was the blob storage details